build.gradle

```
@@ -49,8 +49,11 @@ dependencies {
     annotationProcessor group: "io.kestra", name: "processor", version: kestraVersion
     compileOnly group: "io.kestra", name: "core", version: kestraVersion
     compileOnly group: "io.kestra", name: "script", version: kestraVersion
-}
 
+    // Microsoft Graph SDK - latest stable version
+    implementation "com.microsoft.graph:microsoft-graph:6.54.0"
+    implementation "com.azure:azure-identity:1.15.0"
```

Use the latest please:

```suggestion
    implementation "com.azure:azure-identity:1.18.1"
```